css/responsive/tablet.styl

```diff
@@ -41,11 +41,8 @@ for col in 1..$cols
 // ARCHIVE EDITORIAL
 
 #posts-archive-editorial
-  height: 100%
-  min-height: 600px
   .col
-    height: 100%
-    min-height: 600px
+    min-height: 620px
```

Comment:
ok just to understand how youre approaching this: the logic is such that if the browser height doesnt contain `headerHeight + footerHeight + 620` then the footer then...what happens?